### PR TITLE
Corrections for Ukrainian and Russian languages, attempt № 2

### PR DESCRIPTION
**Pull request by RomanDanilov96**
Since I'm [@bohdangnatiuk](https://github.com/bohdangnatiuk) most likely blocked, as I can no longer submit duplicate Pull requests, and after deleting the repository, I can't even fork it again, I am duplicating my message from a friend's page, who, by the way, also considers it a translation problem.

What's low quality?🤦‍♂️ These're corrections to an existing translation. And the main reason for making them is that the translation of one of the main menu items in Ukrainian, if not the main one at all, is absolutely incorrect. Item: "Налаштування" (in English it's "Preferences", i.e. a synonym of the word "Settings") is translated as: "Вподобання" (the most appropriate English equivalent: "Likes"). This, most likely, is connected with the machine translation, and the person who translated either didn't notice this, or simply doesn't know Ukrainian very well. Would it be acceptable to you if in English instead of "Preferences" you had "Likes" as a menu item? Would it satisfy you, and would you even guess that under the "Likes" item there are actually "Preferences"?! Although the word "Preferences" has several meanings, but in this case, it's the point that is responsible for "Settings", so in Ukrainian it should be "Налаштування", accordingly, that's why in the Russian translation we see the translation of this word as "Настройки", not as "Предпочтения". I've been using FDM for more than one year and at first, I couldn't find the "Preferences" item at all because it's hard to guess that it's called "Likes", only by trial and error. I hoped that the author of the translation himself would fix this over time, but it didn't happen. And since my acquaintances also use FDM based on my recommendations, and I got tired of explaining to them every time that "Likes" are actually "Preferences", so I decided to see if it is possible to make changes to the translation myself. That's all. All other changes are those inaccuracies related to the translation of the word "Preferences", which I noticed during his search for use in the text. And since I noticed, I decided to fix it. Since I also speak Russian, I decided to see if everything is ok in the Russian translation at this moment. Therefore, writing the word "Настроек" instead of "настроек", that is, simply with a capital letter, is just standardization in accordance with other languages. We can continue to elaborate on each of my actions, but I don't see any point in this. Of course, there are probably other inaccuracies in the translation, but I fixed the gross problem that bothered me. If someone else sees other inaccuracies and wants to correct them, that's great. As for me, this is how it works, and the community translation should work. I think everything is clear. Below, I'll attach a screenshot so you can see that the word "Preferences" in Ukrainian indeed has several meanings and in this case, it's translated absolutely incorrectly. And this is really a problem when the main menu item that is used by almost everyone cannot be found because it's not translated correctly.

I'm resending Pull requests, hoping for your understanding and that, on the second attempt, you'll be able to accept these corrections.

### CODE REVIEW

File: main/fdm_ru.po [hunk 1]
```diff
@@ -1010,7 +1010,7 @@ msgid ""
 "Both %1 and QR code are located inside of Preferences page of %2 you're "
 "going to connect to."
 msgstr ""
-"%1 и QR-код находятся на странице настроек %2, к которому вы собираетесь "
+"%1 и QR-код находятся на странице Настроек %2, к которой вы собираетесь "
```
Comment (FreeDownloadManagerTeam) [inline main/fdm_ru.po:1013]: This translation is absolutely wrong.

Comment (RomanDanilov96) [reply main/fdm_ru.po:1013]: This translation is absolutely correct, though it's not exactly a translation but rather just two minor adjustments in two places. Let's go through them step by step:

1. "Настроек" should be written with an uppercase letter, not a lowercase "настроек". It's simply a matter of capitalizing the word. This standardizes it in line with other languages where this term is also capitalized to distinguish it from other words. Here are screenshots from other languages where the word "Настроек/Preferences" is indeed capitalized.






2. The gender of the word "которому" is incorrect. It should be "которой". This can be understood from the context of the entire sentence, assuming you are a native speaker and understand the essence of what is being said. If we were to rely on Google Translate, it often performs poorly in translating sentences with different genders for this word depending on the language. Here's an example:




However, if you look at how the individual word (которой/которому), which is being discussed, is translated, you'll see that the grammatical gender varies between languages, and the reverse translation of the already translated word yields a different result:



DeepL Translator, on the other hand, handles this better and immediately translates with the correct gender for Russian, unlike Ukrainian where the reverse error occurs:



So, in conclusion, if you are not a native speaker, you won't understand what's correct. "К странице настроек..." - "страница" is grammatically feminine, so the question should be "к которой" and not "к которому", because "которому" is grammatically masculine. I hope this clarifies things. Your assertion that "This translation is absolutely wrong" is absolutely absurd for a non-native Russian speaker who doesn't understand the context of the translation as a whole.
